Configurations.md

```diff
@@ -803,6 +803,140 @@ By default this option is set as a percentage of [`max_width`](#max_width) provi
 
 See also [`max_width`](#max_width) and [`width_heuristics`](#width_heuristics)
 
+## `fn_generics_space`
```

Comment:
Just a little bikeshedding on the names, but I'm wondering if something like `fn_sig_generics_spacing` or `fn_generics_spacing` would be more explicit given we're not talking about just one space. What do you think?

Reply:
I agree that "spacing" is better, but "sig" feels redundant since generics are only in the signature. I'm okay with whichever you prefer